File: codigos.js.py/js_html/9c.js
/*Classe Retangulo: Crie uma classe que modele um retangulo:

    Atributos: LadoA, LadoB (ou Comprimento e Largura, ou Base e Altura, a escolher)
    Métodos: Mudar valor dos lados, Retornar valor dos lados, calcular Área e calcular Perímetro;
    Crie um programa que utilize esta classe. Ele deve pedir ao usuário que informe as medidades de um local.
    Depois, deve criar um objeto com as medidas e calcular a quantidade de pisos e de rodapés necessárias para o local.*/
	
var retangulo = {
	base: 5,
	altura: 4,
	
	mudarBase: function(nBase){
		this.base = nBase;
		
		
	},
	mudarAltura: function(nAltura){
		this.altura = nAltura;
		
	},

	valorBase: function(){
		return this.base;
	},
	valorAltura: function(){
		return this.altura;
	},

	area: function(){
		return this.base * this.altura;
	},

	perimetro: function(){
		return 2 * (this.base + this.altura);
	}


}

document.write("Base do Retangulo: ", (retangulo.valorBase()));
document.write(" => Altura do Retangulo: ", (retangulo.valorAltura()));
document.write(" => Área do Retangulo: ", (retangulo.area()));
document.write(" => Perímetro do Retangulo: ", (retangulo.perimetro()));
retangulo.mudarBase(3);
retangulo.mudarAltura(1);
document.write(" => Nova base do Retangulo: ", (retangulo.valorBase()));
document.write(" => Nova altura do Retangulo: ", (retangulo.valorAltura()));

File: codigos.js.py/scripts_py/9c.py
'''Classe Retangulo: Crie uma classe que modele um retangulo:

    Atributos: LadoA, LadoB (ou Comprimento e Largura, ou Base e Altura, a escolher)
    Métodos: Mudar valor dos lados, Retornar valor dos lados, calcular Área e calcular Perímetro;
    Crie um programa que utilize esta classe. Ele deve pedir ao usuário que informe as medidades de um local.
    Depois, deve criar um objeto com as medidas e calcular a quantidade de pisos e de rodapés necessárias para o local.'''
class retangulo:
    def __init__(self, base, altura):
        self.base = base
        self.altura = altura
    
    def mudar_base(self, n_base):
        self.base = n_base

    def mudar_altura(self, n_altura):
        self.altura = n_altura

    def valor_base(self):
        return self.base

    def valor_altura(self):
        return self.altura

    def area(self):
        return self.base * self.altura

    def perimetro(self):
        return 2 * (self.base + self.altura)

r = retangulo(3, 2)
print("\nValor da base =  ", r.valor_base())
print("Valor da altura =  ", r.valor_altura())
print("Área do retângulo = ", r.area())
print("Perímetro do retângulo =  ", r.perimetro())
r.mudar_base(4.3)
print("Novo valor da base =  ", r.valor_base())
r.mudar_altura(2.5)
print("Novo valor da altura =  ", r.valor_altura())
print("")


	
	